### PR TITLE
CLIMATE-715 Adding a new demo tab along with the evaluate and result.

### PR DESCRIPTION
**Pull request by Omkar20895**
- Added a new demo tab.
- The tab includes a demo video of how the web-app works.
- Added a new footer, this prevents the app views from ending abruptly in the web-app.
- The footer contains important links like link to the OCW website and also to the issue tracker.

### CODE REVIEW

File: ocw-ui/frontend/app/index.html [hunk 1]
```diff
@@ -143,5 +143,21 @@ <h5 id="OCW-powered">Powered by Apache Open Climate Workbench</h5>
...
     <!-- grunt wiredep wasn't properly linking this. Had to add manually -->
     <script src="bower_components/angular-ui-date/src/date.js"></script>
+
+    <!-- Footer for the Web-App -->
+    <footer class="footer" align="center">
+      <div class="container">
+        <hr>
+        <p>
+          Apache Open Climate Workbench Software . 
+          <a href="https://climate.apache.org/" target="_blank">About Us</a> . <a href="https://issues.apache.org/jira/browse/CLIMATE/?selectedTab=com.atlassian.jira.jira-projects-plugin:issues-panel" target="_blank">Report an issue</a> .
```
Comment (lewismc) [inline ocw-ui/frontend/app/index.html]: The URL here can be abbreviated to https://issues.apache.org/jira/browse/CLIMATE or a mailto link which points to dev@climate.apache.org. Non technical people may be more likely to use the mailto link.

File: ocw-ui/frontend/app/index.html [hunk 2]
```diff
@@ -143,5 +143,21 @@ <h5 id="OCW-powered">Powered by Apache Open Climate Workbench</h5>
...
+          <br>
+          Copyright © 2013 The Apache Software Foundation, Licensed under the 
+
```
Comment (lewismc) [inline ocw-ui/frontend/app/index.html]: Can remove blank line

**Comment by lewismc** [comment]
@Omkar20895 this is a pretty cool patch, I was thinking about getting a test instance of the WebApp up and hosted at the ASF cf. https://issues.apache.org/jira/browse/CLIMATE-689

**Comment by Omkar20895** [comment]
Hello Lewis, 
I have updated the pull request, please take a look at it.
I think the issue addressed at CLIMATE-689 is a splendid idea and we must definitely host the web-app publicly, this reduces the burden of downloading the whole application repository on the user. At present I am working on CLIMATE-383 which is also a good idea and will work on this one too simultaneously.
Thanks.

**Pull request merged**